runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/foreign_investments.rs

```diff
@@ -112,6 +112,10 @@ mod same_currencies {
 			let currency_id = AUSD_CURRENCY_ID;
 			let currency_decimals = currency_decimals::AUSD;
 
+			let sending_domain_locator = Domain::convert(DEFAULT_DOMAIN_ADDRESS_MOONBEAM.domain());
+			Tokens::mint_into(AUSD_CURRENCY_ID, &sending_domain_locator, AUSD_ED).unwrap();
+			Tokens::mint_into(AUSD_CURRENCY_ID, &investor, AUSD_ED).unwrap();
```

Comment:
Add `ED` for the sending domain account and for the investor

Reply:
UPDATE: I've also added `AUSD_ED` to the pool_account, but the `ExistentialDeposit` error still appears.